test/units/module_utils/common/arg_spec/test_aliases.py

```diff
@@ -95,6 +95,11 @@ def test_aliases(arg_spec, parameters, expected, deprecation, warning):
     assert isinstance(result, ValidationResult)
     assert result.validated_parameters == expected
     assert result.error_messages == []
+    assert result._aliases == {
+        alias: param
+        for param, value in arg_spec.items()
+        for alias in value.get("aliases", [])
+    }
```

Comment:
This needs to be rewritten to support Python 2.6.

Reply:
On it.

Reply:
Should be resolved now. Should I open a PR for that in `devel` or just leave it in this backport?

Reply:
It's not a problem for devel (or stable-2.13) since we dropped support for Python 2.6 in those branches.